### ISSUE TITLE
New game: To cities.

### ISSUE THREAD

**Issue by kirillPro2020** [Monika-After-Story/MonikaModDev#5608]
Why not play with Monica  to cities.
( Still know the rules, huh?).

**Comment by AsuraShun** [comment]
What game is that?

**Comment by kirillPro2020** [comment]
This is a game where one player calls the city. Anyone he wants. And the second should call the city based on the last letter of the named city of the firsr player. Then they change.
( For example: New York means the second player needs to name the city with the letter k.
It turns out Kent).

**Comment by kisik21** [comment]
That's a game I would play. This Wikipedia article might be useful in creating the dataset for cities: https://en.wikipedia.org/wiki/Lists_of_cities_and_towns

Also maybe Monika could use a simple dictionary to play something similar to shiritori - just use words instead of city names.

**Comment by jmwall24** [comment]
Looking at this, would likely have to limit to cities with over 100k people in it. There's the most comprehensive list for that and also you go lower than that the list gets even huge than it already is.

**Comment by kisik21** [comment]
Sounds fine, but maybe let Monika warn the player that "they only covered relatively big cities in their geography classes"

**Comment by jmwall24** [comment]
Yeah, we can just specify the rules are only cities with over 100000 people. The list of every city, town, village would be like 2.5M lol

**Comment by kirillPro2020** [comment]
I think that it will be  enough, to indicate cities with millionaires, there are about 350 of them.
(In this game, only cities are called)

**Comment by jmwall24** [comment]
I just populated the list of cities with over 100l people and it's 4292 long, so manageable.

**Comment by kirillPro2020** [comment]
Hmm, I found 4809 cities with a population of 100k people.
(it's still a lot).

**Comment by jmwall24** [comment]
can i see where you got that list please?

**Comment by kirillPro2020** [comment]
I found it at a foreign source. I can drop the link if necessary.

**Comment by jmwall24** [comment]
Please do

**Comment by kirillPro2020** [comment]
Here: https://www.wikiznanie.ru/wikipedia/index.php/%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%B3%D0%BE%D1%80%D0%BE%D0%B4%D0%BE%D0%B2_%D1%81_%D0%BD%D0%B0%D1%81%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5%D0%BC_%D0%B1%D0%BE%D0%BB%D0%B5%D0%B5_100_%D1%82%D1%8B%D1%81%D1%8F%D1%87_%D0%B6%D0%B8%D1%82%D0%B5%D0%BB%D0%B5%D0%B9

**Comment by jmwall24** [comment]
Thanks. I think we'll use the list I found, it's newer data, sorted better and I don't have to translate it lol.  I appreciate it though!

**Comment by AsuraShun** [comment]
This game actually sounds fun

**Comment by kirillPro2020** [comment]
I have a question regarding the list of city names.
Is it a plain text file or something else?

**Comment by ThePotatoGuy** [comment]
so shiritori but with cities

**Comment by ThePotatoGuy** [comment]
[closing until we look into it]

**Comment by jmwall24** [comment]
the linked PR is the same game, just different words would be used (cities here, normal words for the other) so reopening this as in-progress.
